Clang CodeBuild configuration gets error in the build step

The Clang CodeBuild builds are getting the following error:

```
[Container] 2020/11/04 12:34:39 Running command ecbuild /src_repo/ioda-converters
Found CMake version 3.17.2

cmake -DCMAKE_MODULE_PATH=/usr/local/share/ecbuild/cmake /src_repo/ioda-converters 

CMake Error at /usr/local/share/cmake-3.17/Modules/CMakeDetermineCXXCompiler.cmake:48 (message):
  Could not find compiler set in environment variable CXX:

  mpiicpc.

Call Stack (most recent call first):
  CMakeLists.txt:12 (project)


CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_Fortran_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
See also "/build_container/CMakeFiles/CMakeOutput.log".
```

It looks like the wrong mpi wrapper is being used for the CXX compiler. (`mpiicpc` is the Intel wrapper, correct?)

--- a/buildspec_clang.yml
+++ b/buildspec_clang.yml
@@ -32,7 +32,7 @@ phases:
       - git lfs install
       - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
-      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
+      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in ioda-converters repo"
       - cd $CODEBUILD_SRC_DIR
 
 

--- a/buildspec_clang.yml
+++ b/buildspec_clang.yml
@@ -30,7 +30,7 @@ phases:
       - echo "CODEBUILD_SOURCE_VERSION=${CODEBUILD_SOURCE_VERSION}"
 
       - git lfs install
-      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda/ioda-converters
+      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
       - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
       - cd $CODEBUILD_SRC_DIR

--- a/buildspec_gnu.yml
+++ b/buildspec_gnu.yml
@@ -60,7 +60,7 @@ phases:
       - git lfs install
       - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
-      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
+      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in ioda-converter repo"
       - cd $CODEBUILD_SRC_DIR
 
 

--- a/buildspec_gnu.yml
+++ b/buildspec_gnu.yml
@@ -58,7 +58,7 @@ phases:
         && echo MPI setup for Docker done"
 
       - git lfs install
-      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda/ioda-converters
+      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
       - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
       - cd $CODEBUILD_SRC_DIR

--- a/buildspec_intel.yml
+++ b/buildspec_intel.yml
@@ -54,7 +54,7 @@ phases:
       - echo "CODEBUILD_GIT_BRANCH=${CODEBUILD_GIT_BRANCH}"
       - echo "CODEBUILD_SOURCE_VERSION=${CODEBUILD_SOURCE_VERSION}"
 
-      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda/ioda-converters
+      - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
       - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
       - cd $CODEBUILD_SRC_DIR

--- a/buildspec_intel.yml
+++ b/buildspec_intel.yml
@@ -56,7 +56,7 @@ phases:
 
       - git clone https://$GIT_USER:$GIT_PASS@github.com/jcsda-internal/ioda-converters
       - cd ioda-converters
-      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in saber-bundle repo"
+      - git checkout $CODEBUILD_GIT_BRANCH || echo "No branch named $CODEBUILD_GIT_BRANCH in ioda-converters repo"
       - cd $CODEBUILD_SRC_DIR
 
       - mv ioda-converters /jcsda/

